projects/packages/waf/src/brute-force-protection/class-shared-functions.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * Class for functions shared by the Brute force protection module and its related json-endpoints
```

Comment:
```suggestion
 * Class for functions shared by the Brute Force Protection feature and its related json-endpoints
```

Since we've pulled the functionality into the package I think we want to move away from calling it a "module".